### ISSUE TITLE
BNC vs BCE with logit loss

### ISSUE THREAD

**Issue by kordjamshidi** [HLR/DomiKnowS#356]
Where is the accurate explanation of these two losses in which problems which one should be used. Can we have this in the documentation?

**Comment by hfaghihi15** [comment]
BNC Loss and BCE loss are the losses defined in the PyTorch library, we have just wrapped them here. 
BCE: https://pytorch.org/docs/stable/generated/torch.nn.BCELoss.html
CrossEntropy: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html
